packages/cli/src/index.ts

```diff
@@ -68,14 +120,12 @@ try {
   }
 
   await cli.runExit(proc, panfactumContext);
-
-  // todo: move to finally block
-  await phClient.shutdown()
 } catch(error: unknown) {
```

Comment:
This is swallowing the exit code.

This also doesn't print the error out according to our error formatting.